geometry/list-transformations-by-suitability/README.md

```diff
@@ -27,6 +27,22 @@ If the selected transformation is not usable (has missing grid files) then an er
 * GeometryEngine
 * TransformationCatalog
 
+## Additional information
+
+Some transformations aren't available until transformation data is provided.
+
+This sample uses a `GeographicTransformation`, which extends the `DatumTransformation` class. As of 100.9, ArcGIS Runtime also includes a `HorizontalVerticalTransformation`, which also extends `DatumTransformation`. The `HorizontalVerticalTrasnformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
```

Comment:
It looks like in our sample we don't use the `GeographicTransformation` class: just the `DatumTransformation` class. Whenever I do a ReadMe update which contains new classes, I always do a quick "Find..." within the sample just to sanity check if we use the class from the generic sample description or not: samples can vary slightly between SDKs and also in name, so it's a good habit to get into! 

What is the purpose of the addition of this additional information here: is it intended that this sample use the `HorizontalVerticalTransformation`? Does the code require updating? If not then we can just reword to avoid specifically saying that our samples uses the `GeographicTransformation: but you should probably double check the wording with the author of the common sample readme to be sure this still delivers the message intended. 

```suggestion
As of 100.9, ArcGIS Runtime also includes a `HorizontalVerticalTransformation`, which extends `DatumTransformation`. The `HorizontalVerticalTrasnformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
```